pages/workspace/workspace.ejs

```diff
@@ -7,6 +7,32 @@
 </head>
 
 <body class="d-flex flex-column h-100">
+  <div class="modal fade" id="rebootModal" tabindex="-1" aria-labelledby="rebootModalLabel" aria-hidden="true">
+    <div class="modal-dialog">
+      <div class="modal-content">
+        <div class="modal-header">
+          <h1 class="h5 modal-title" id="rebootModalLabel">Reboot the workspace</h1>
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div>
+        <div class="modal-body">
+          <p>Are you sure you want to reboot the virtual machine?<p>
+          <ul class="fa-ul">
+            <li><span class="fa-li"><i class="fas fa-window-restore" aria-hidden="true"></i></span>Your workspace will reboot into this same tab.</li>
+            <li><span class="fa-li"><i class="fas fa-save" aria-hidden="true"></i></span>Your files will remain intact through the reboot.</li>
+          </ul>
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
```

Comment:
I think we are generally using `Cancel`? https://github.com/PrairieLearn/PrairieLearn/blob/d59233ec111d0b0daff2c06574d40acdafaf8e20/pages/instructorAssessmentInstances/instructorAssessmentInstances.ejs#L40

Reply:
fixed in 25f9be8c3f5be30d776d8ca12ebf13535aa037cc